tools/cli/test/data/split/dev/openapi-v2-2023-01-01.json

```diff
@@ -1,15 +1,4 @@
 {
-  "info": {
```

Comment:
I needed to update the specifications used for the e2e split command. The issue was that, due to the filters applied, the generated specs were missing unused schemas. This caused the test to fail because the generated specs were not identical to the expected ones.